gradio/components.py

```diff
@@ -1257,7 +1259,10 @@ def __init__(
             value=value,
             **kwargs,
         )
-        self.cleared_value = self.value
+
+        self.cleared_value = (
+            self.value if self.value is not None else [] if multiselect is True else ""
```

Comment:
Might be helpful for readability
```suggestion
        self.cleared_value = (
            self.value if self.value is not None else ([] if multiselect is True else "")
```